src/solver/optimisation/constraints/ConstraintBuilder.cpp

```diff
@@ -58,6 +58,25 @@ ConstraintBuilder& ConstraintBuilder::DispatchableProduction(unsigned int index,
                                                              int delta)
 {
     AddVariable(GetVariableManager(offset, delta).DispatchableProduction(index), coeff);
+    // logs.info() << "dispatchable production idx " << GetVariableManager(offset, delta).DispatchableProduction(index);
+    return *this;
```

Comment:
remove debug comments

Reply:
fixed in commit [624f2ce](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/624f2ce5436532ef019be957135c5cbb154c5a60)